src/lib/utils/time.test.ts

```diff
@@ -0,0 +1,38 @@
+import { describe, expect, it } from 'vitest'
+
+import { formatSecondsToDuration } from './time'
+
+describe('formatSecondsToDuration', () => {
```

Comment:
At this point pretty useless as this just tests dayjs does its dayjs thing but meh - if we ever get to nicely working thresholds, then this would actually give us confidence our thresholds work as they should